spec/factories/songs.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :song do
+    name 'Dirty Diana'
+    artist 'Michael Jackson'
+    spotify_uri { SecureRandom.hex(10) }
+  end
+end
```

Comment:
If using SublimeText, check this preference setting:

```
    // Set to true to ensure the last line of the file ends in a newline
    // character when saving
    "ensure_newline_at_eof_on_save": true,
```

By setting it to `true`, a newline will always be created and Github won't issue the warning.

But, you do have to open up and save the file again to get the newline there.